### ISSUE TITLE
fix:  improve site Accessibility

### ISSUE THREAD

**Issue by shseok** [shseok/mztrees#8]
1. Buttons do not have an accessible name
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.
![image](https://github.com/shseok/mztrees/assets/80726955/b222a56e-98c5-4730-ab81-4797a216c15b)

3. Links do not have a discernible name
Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users.
![image](https://github.com/shseok/mztrees/assets/80726955/71a319ed-bc91-4e63-b675-12cea0f1ed49)
